os/android.toolchain.cmake

```diff
@@ -218,7 +218,7 @@ set( CMAKE_SHARED_LIBRARY_RUNTIME_C_FLAG "" )
 set( CMAKE_SKIP_RPATH TRUE CACHE BOOL "If set, runtime paths are not added when using shared libraries." )
 
 # NDK search paths
-set( ANDROID_SUPPORTED_NDK_VERSIONS ${ANDROID_EXTRA_NDK_VERSIONS} -r10d -r10c -r10b -r10 -r9d -r9c -r9b -r9 -r8e -r8d -r8c -r8b -r8 -r7c -r7b -r7 -r6b -r6 -r5c -r5b -r5 "" )
+set( ANDROID_SUPPORTED_NDK_VERSIONS ${ANDROID_EXTRA_NDK_VERSIONS} -r11c -r11b -r11 -r10e -r10d -r10c -r10b -r10 -r9d -r9c -r9b -r9 -r8e -r8d -r8c -r8b -r8 -r7c -r7b -r7 -r6b -r6 -r5c -r5b -r5 "" )
```

Comment:
It would be nice to have `android-ndk-r11c-api-19-armeabi-v7a-neon` testing on Travis (there is `android-ndk-r10e-api-19-armeabi-v7a-neon` currently). For this needed:
* Patch https://github.com/hunter-packages/android-ndk/blob/master/create-android-ndk.py script so it support ndk version option
* add  SHA1 for `android-ndk-*` binaries

After this update I can upload new `*.tar.gz` NDK archives for CI testing to https://github.com/hunter-packages/android-ndk/releases, next steps:
* update https://github.com/ruslo/polly/blob/master/bin/install-ci-dependencies.py
* update .travis.yml